doc/api/corepack.md

```diff
@@ -12,10 +12,9 @@ added:
 
 > Stability: 1 - Experimental
 
-Documentation for this tool can be found on the [Corepack repository][].
+**Corepack will no longer be distributed starting with Node.js v25.**
 
-Despite Corepack being distributed with default installs of Node.js, the package
-managers managed by Corepack are not part of the Node.js distribution, and
-Corepack itself will no longer be distributed with Node.js 25+.
+Users currently depending on the bundled `corepack` module from Node.js
+should switch to using the userland-provided [corepack][] module instead.
```

Comment:
Can we word it in a way it sounds like proposing an alternative instead of giving an advice?
```suggestion
can switch to using the userland-provided [corepack][] module instead.
```

Reply:
```suggestion
can switch to using the userland-provided [corepack][] module.
```

Reply:
Done.